Commandant/ArgumentParser.swift

```diff
@@ -40,6 +46,10 @@ extension RawArgument: Printable {
 
 		case let .Value(value):
 			return "\"\(value)\""
+
+		case let .Flag(flags):
+			let combined = join("", map(flags) { String($0) })
```

Comment:
I think you can replace this whole expression with `String(flags)`, because there's a constructor which accepts a `SequenceType`.